### PR TITLE
BM-388 improve traveller data structure.

### PR DESCRIPTION
**Pull request by greg-fti**

### CODE REVIEW

**Review by EyesOnlyNet — COMMENTED** [review]
the adjustment of the test file is missing - without that no one can test your change

File: index.d.ts [hunk 1]
```diff
@@ -132,7 +141,7 @@ declare namespace bookingManagerConnector {
         renter: CarRenter;
         pickUp: VehicleLocation;
         dropOff: VehicleLocation;
-        services: Array<string>;
+        services: Array<string> | Array<any>;
```
Comment (EyesOnlyNet) [inline index.d.ts]: for this version `Array<any>` is not allowed

File: index.d.ts [hunk 2]
```diff
@@ -188,6 +197,7 @@ declare namespace bookingManagerConnector {
         toDate: string;
         location: string;
         hotel: string;
+        description?: string
```
Comment (EyesOnlyNet) [inline index.d.ts]: why do we need a "description" for the RoundTripRoute? do we need it in the BM? than you have to document it in the ReadMe too.

File: tests/manual/index.html [hunk 3]
```diff
@@ -194,21 +194,24 @@ <h2>Select the product type you want to transfer to the BM:</h2>
                         <fieldset class="col my-1">
                             <legend>travellers</legend>
                             <div data-form-field data-label="type" data-name="travellers.0.type" data-value="female"></div>
-                            <div data-form-field data-label="name" data-name="travellers.0.name" data-value="jane doe"></div>
+                            <div data-form-field data-label="name" data-name="travellers.0.firstName" data-value="jane"></div>
```
Comment (EyesOnlyNet) [inline tests/manual/index.html]: maybe you should change the text of the label too!?

File: index.d.ts [hunk 4]
```diff
@@ -191,18 +201,23 @@ declare namespace bookingManagerConnector {
         latitude: number | string;
         longitude: number | string;
     }
+    export interface RoundTripRouteTransfer {
```
Comment (EyesOnlyNet) [inline index.d.ts:204]: you should document it in the readMe

**Review by EyesOnlyNet — APPROVED** [review]

File: index.d.ts (hunk 4 above)
Comment (greg-fti) [reply index.d.ts:204]: Done.

**Comment by EyesOnlyNet** [comment]
can you also add an example for "RoundTripRouteTransfer" to the test file?!

File: index.d.ts [hunk 5]
```diff
@@ -181,8 +191,8 @@ declare namespace bookingManagerConnector {
         name: string;
         quantity: number | string;
     }
-    export interface RoundTripRoute {
-        type: RoundTripRouteType | string;
+    export interface RoundTripRouteAccomodation {
```
Comment (EyesOnlyNet) [inline index.d.ts]: google say "accommodation" with double "m" is correct

**Review by EyesOnlyNet — APPROVED** [review]

**Pull request merged**